extensions/sources/sql/ext/sqlLookup.go

```diff
@@ -62,29 +68,7 @@ func (s *sqlLookupSource) Configure(datasource string, props map[string]interfac
 func (s *sqlLookupSource) Lookup(ctx api.StreamContext, fields []string, keys []string, values []interface{}) ([]api.SourceTuple, error) {
 	ctx.GetLogger().Debug("Start to lookup tuple")
 	rcvTime := conf.GetNow()
-	query := "SELECT "
-	if len(fields) == 0 {
-		query += "*"
-	} else {
-		for i, f := range fields {
-			if i > 0 {
-				query += ","
-			}
-			query += f
-		}
-	}
-	query += fmt.Sprintf(" FROM %s WHERE ", s.table)
-	for i, k := range keys {
-		if i > 0 {
-			query += " AND "
-		}
-		switch v := values[i].(type) {
-		case string:
-			query += fmt.Sprintf("`%s` = '%s'", k, v)
-		default:
-			query += fmt.Sprintf("`%s` = %v", k, v)
-		}
-	}
+	query := s.buildQuery(fields, keys, values)
```

Comment:
Is it better/possible to use the query builder which already consider different db.